README.md

```diff
@@ -92,11 +92,29 @@ If you use Swift Package Manager, simply add Alicerce as a dependency of your pa
 1. If linking against an Application target, ensure the framework gets copied into the bundle. If linking against a Framework target, 
 the application linking to it should also include Alicerce.
 
+## Setup
+
+Setting up the project for development is simple:
+
+1. Clone the repository.
+2. Retrieve the project dependencies using one of the following commands from the project root directory:
+    * if you have Carthage installed: `carthage checkout`
+    * or: `git submodule update --init --recursive`
+3. Open `Alicerce.xcworkspace`
+4. Build `Result-iOS` scheme
+4. Build `Alicerce` scheme
+
 ## Contributing 🙌
 
-See [CONTRIBUTING](https://github.com/Mindera/Alicerce/blob/master/CONTRIBUTING.md).
+See [CONTRIBUTING].
+
+## License
+
+Alicerce is Copyright (c) 2016 Mindera and is available under the MIT License. It is free software, and may be redistributed under the terms specified in the [LICENSE] file.
```

Comment:
Shouldn't the year be 2018, or a range? (2016-2018)? 💄

Reply:
Updated it to be a range. ✨